sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/_connection.py

```diff
@@ -644,8 +653,7 @@ def _process_incoming_frame(self, channel, frame):  # pylint:disable=too-many-re
         except KeyError:
             return True  # TODO: channel error
 
-    def _process_outgoing_frame(self, channel, frame):
-        # type: (int, NamedTuple) -> None
+    def _process_outgoing_frame(self, channel: int, frame: NamedTuple) -> None:
```

Comment:
If we type this as `NamedTuple` we're going to have issues in the `encode` method where we access things like the `_code` property.